flink-runtime/src/main/java/org/apache/flink/runtime/entrypoint/component/AbstractUserClassPathJobGraphRetriever.java

```diff
@@ -51,7 +51,7 @@ protected AbstractUserClassPathJobGraphRetriever(@Nullable final File jobDir) th
 		}
 	}
 
-	protected List<URL> getUserClassPaths() {
+	public List<URL> getUserClassPaths() {
```

Comment:
If you are making it public for testing, please annotate it with @VisibleForTesting.
However, I think it's not good to test what you is not changed in this PR.